tests/integration-tests-base/pom.xml

```diff
@@ -63,6 +63,8 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-surefire-plugin</artifactId>
         <configuration>
+          <!-- only run tests when -DintegrationTests is specified //-->
+          <skipTests>true</skipTests>
```

Comment:
Wouldn't it be better to use failsafe and do not run on test goal but with verify?

Reply:
good point. +1

Reply:
I looked a bit at failsafe. Does verify need to be called explicitly? As opposed to test which runs with package and install?

Reply:
It has been designed expecially for integration tests I suppose.

Even using -DintegrationTests needs explicit request from user.

Maybe it is better to:
1) use directly a profile, -PintegrationTests (you can activate more than one profile on the command line)
2) use failsafe and so the verify command

To be honest I have never used failsafe but as far as I know it is the 'maven way'.

Reply:
there is a profile "integrationTests", so it can be activated by -P, I created the activate by system property so that docker would also be activated.

I _want_ it to be explicit for the user to run these tests. I don't want users to have to have docker installed to be able to get through a "mvn install".